modules/math/src/main/java/com/opengamma/strata/math/impl/statistics/descriptive/QuantileCalculationMethod.java

```diff
@@ -63,34 +43,14 @@ public double quantileFromUnsorted(double level, DoubleArray sample) {
    * If index value computed from the level is outside of the sample data range, the nearest data point is used, i.e., 
    * quantile is computed with flat extrapolation.
    * <p> 
-   * The sample observations are sorted from the smallest to the largest. 
-   * 
-   * @param level  the quantile level
-   * @param sortedSample  the sample observations
-   * @return the quantile estimation
-   */
-  public double quantileWithExtrapolationFromSorted(double level, DoubleArray sortedSample) {
-    return quantile(level, sortedSample, true);
-  }
-
-  /**
-   * Compute the quantile estimation.
-   * <p>
-   * The quantile level is in decimal, i.e. 99% = 0.99 and 0 < level < 1 should be satisfied.
-   * This is measured from the bottom, that is, the quantile estimation with the level 99% corresponds to 
-   * the smallest 99% observations and 1% of the observation are above that level.
-   * <p>
-   * If index value computed from the level is outside of the sample data range, the nearest data point is used, i.e., 
-   * quantile is computed with flat extrapolation. 
-   * <p> 
-   * The sample observations are supposed to be unsorted, the first step is to sort the data.
+   * The sample observations are supposed to be unsorted.
```

Comment:
Please add javadoc mentioning that `indices` in `QuantileResult` are based on the original unsorted array.